From a69009ce0be3fef24fa9989bbfa945896c9fcda7 Mon Sep 17 00:00:00 2001
From: Gilles Filippini <pini@debian.org>
Date: Sun, 23 Sep 2012 16:31:57 +0200
Subject: [PATCH] Fix build error when cross building with g++-4.4

Fix error "format not a string literal and no format arguments".
---
 src/declarative/debugger/qdeclarativedebugserver.cpp |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/declarative/debugger/qdeclarativedebugserver.cpp b/src/declarative/debugger/qdeclarativedebugserver.cpp
index 5e2b899..d9a85a2 100644
--- a/qtopiacore/qt/src/declarative/debugger/qdeclarativedebugserver.cpp
+++ b/qtopiacore/qt/src/declarative/debugger/qdeclarativedebugserver.cpp
@@ -207,9 +207,9 @@ QDeclarativeDebugServer *QDeclarativeDebugServer::instance()
                 }
 
             } else {
-                qWarning(QString::fromAscii("QDeclarativeDebugServer: Ignoring \"-qmljsdebugger=%1\". "
+                qWarning() << QString::fromAscii("QDeclarativeDebugServer: Ignoring \"-qmljsdebugger=%1\". "
                                             "Format is -qmljsdebugger=port:<port>[,block]").arg(
-                             appD->qmljsDebugArgumentsString()).toAscii().constData());
+                             appD->qmljsDebugArgumentsString()).toAscii().constData();
             }
         }
 #else
-- 
1.7.10.4

